ml_dtypes/include/mxfloat.h

```diff
@@ -0,0 +1,380 @@
+/* Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef ML_DTYPES_MXFLOAT_H_
+#define ML_DTYPES_MXFLOAT_H_
+
+// Microscaling (MX) floating point formats, as described in
```

Comment:
Perhaps note that these are just the raw data types, without scales.

Reply:
Added a comment.